src/Monolog/Handler/RotatingFileHandler.php

```diff
@@ -80,6 +85,10 @@ protected function write(array $record)
             $this->mustRotate = !file_exists($this->url);
         }
 
+        if (null === $this->mustSymlink) {
+            $this->mustSymlink = !is_link($this->filename);
+        }
```

Comment:
I don't really get the point here, if the current day's file is not a link it gets symlinked, which means the feature isn't really optional. That might be fine, but what happens once it got rotated, the symlink will point to an old one?